intake/src/main/resources/processor_bootstrap.sh

```diff
@@ -3,13 +3,13 @@
 LDSC_ROOT=/mnt/var/intake
 
 # install to the root directory
-mkdir -p "$LDSC_ROOT"
+sudo mkdir -p "$LDSC_ROOT"
```

Comment:
Seemingly an odd quirk of switching EMR images is that version 6 needs sudo peppered everywhere in any bash script.